plugins/UsersManager/Model.php

```diff
@@ -141,6 +141,11 @@ public function getUsersLoginWithSiteAccess($idSite, $access)
         return $logins;
     }
 
+    public function tokenTableExists()
+    {
+        return !!$this->getDb()->fetchOne(sprintf("SHOW TABLES LIKE '%s';", $this->tokenTable));
```

Comment:
fyi you could probably use `DbHelper::getTablesInstalled` for this? Or maybe add a new method to DbHelper?

Reply:
Ok. I'll move that to DbHelper. There was a method `tableExists`, but it has been removed as it was unused...